lib/wallet/integration/src/Test/Integration/Scenario/API/Shelley/Migrations.hs

```diff
@@ -264,8 +267,8 @@ spec = describe "SHELLEY_MIGRATIONS" $ do
                 >>= flip verify
                 [ expectField (#balance . #available . #getQuantity)
                     (.> 0)
-                , expectField (#assets . #available . #getApiT)
-                    ((.> 0) . TokenMap.size)
+                , expectField (#assets . #available)
+                    ((.> 0) . length . toList)
```

Comment:
it has `IsList` instance now, correct? :1st_place_medal:

Reply:
> it has `IsList` instance now, correct? 🥇

Yes, indeed!